hack/revert-ci-operator.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+kubectl config use-context api.ci
```

Comment:
It's a nit but let's make the script depend on just `oc`. Either `oc config use-context`, or just run the oc with `--context` in the commands below.

Reply:
Done